spyder/plugins/pythonpath/plugin.py

```diff
@@ -34,7 +34,7 @@ class PythonpathManager(SpyderPluginV2):
     CONF_SECTION = NAME
     CONF_FILE = False
 
-    sig_pythonpath_changed = Signal(object, object)
+    sig_pythonpath_changed = Signal(object, bool)
```

Comment:
Please update the description of the parameters in the docstring below: remove `old_path_dict` and change `new_path_dict` to reflect that it is now a list of strings.